src/uu/true/src/true.rs

```diff
@@ -19,5 +19,5 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
 }
 
 pub fn uu_app() -> App<'static, 'static> {
-    App::new(executable!())
+    App::new(util_name!())
```

Comment:
not sure if we need to fix this now, but since `true` does not override `usage` we don't get the right usage string when invoking `true` with the multicall binary.